blog-article.html

```diff
@@ -0,0 +1,317 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta
+      name="description"
+      content="The first huge metaverse arena to arrange live sports, hang out and even make bets and wagers"
+    />
+    <title>Blog Article - BINABOX</title>
+  </head>
+  <body>
+    <!-- Header -->
+    <header>
+      <a href="./index.html" class="logo-block">
+        <img src="./img/img-logo/logo.svg" alt="Binabox logo" />
+        BINABOX
+      </a>
+      <!-- Navigation Menu -->
+      <nav class="nav-menu">
+        <ul class="nav-menu-items">
+          <li class="nav-menu-item">
+            <a href="index.html">HOME</a>
+          </li>
+
+          <li class="nav-menu-item">
+            <a href="./about.html">ABOUT</a>
+          </li>
+
+          <li class="nav-menu-item">
+            <a href="./index.html#roadmap-block">ROADMAP</a>
+          </li>
+
+          <li class="nav-menu-item">
+            <a href="./catalog.html">COLLECTION</a>
+          </li>
+
+          <li class="nav-menu-item">
+            <a href="./faqs.html">FAQS</a>
+          </li>
+
+          <li class="nav-menu-item">
+            <a href="#" class="dropdown">PAGES</a>
+            <ul class="dropdown-content">
+              <li>
+                <a href="./catalog.html">NFT Catalog</a>
+              </li>
+              <li>
+                <a href="./blog.html">Blog</a>
+              </li>
+              <li>
+                <a href="./contact-us.html">Contact Us</a>
+              </li>
+            </ul>
+          </li>
+        </ul>
+      </nav>
+
+      <nav class="btn-box">
+        <button class="switch-mode">
+          <img src="./img/btn-icons/switch.png" alt="" />
+        </button>
+
+        <a href="https://discord.com/" class="header-link" target="_blank">
+          <img src="./img/socialntw-icons/discord-btn.png" alt="" />
+          DISCORD
+        </a>
+
+        <button class="header-btn">
+          <img src="./img/btn-icons/wallet.png" alt="" />
+          CONNECT
+        </button>
+      </nav>
+    </header>
+
+    <!-- Main -->
+    <main>
+      <!-- Breadcrumbs -->
+      <section class="breadcrumbs">
+        <h1>BLOG DETAIL</h1>
+        <ul>
+          <li><a href="./index.html">HOME</a></li>
+          <li><a href="./blog.html">OUR BLOG</a></li>
+          <li><a href="#blog-detail">BLOG DETAIL</a></li>
+        </ul>
+      </section>
+
+      <section class="blog-article" id="blog-detail">
+        <img
+          src="./img/img-blog/img-blog-article_main.png"
+          alt="The man standing infront of the giant green dragon"
+        />
+        <h2>Enior Apple Employee Alleges Sexism At</h2>
+        <p>GAMING</p>
+        <ul>
```

Comment:
It's not a list. 
A list is an enumeration of homogeneous elements with the same meaning and styling. This means that any number of items can be added to the list, and they should all look the same.